tutorials/clinical/10_ieeg_localize.py

```diff
@@ -111,89 +110,43 @@ def plot_overlay(image, compare, title, thresh=None):
-                        moving_affine=CT_orig.affine,
-                        static_affine=T1.affine,
-                        between_affine=None)
+                        moving_affine=CT_clean.affine,
+                        static_affine=T1.affine)
 plot_overlay(T1, CT_resampled, 'Unaligned CT Overlaid on T1', thresh=0.95)
+del CT_resampled
 
 ###############################################################################
 # Now we need to align our CT image to the T1 image.
 #
-# .. code-block:: python
-#
-#     # normalize intensities
-#     mri_to = T1.get_fdata().copy()
-#     mri_to /= mri_to.max()
-#     ct_from = CT_orig.get_fdata().copy()
-#     ct_from /= ct_from.max()
-#
-#     # downsample for speed
-#     zooms = (5, 5, 5)
-#     mri_to, affine_to = reslice(
-#         mri_to, affine=T1.affine,
-#         zooms=T1.header.get_zooms()[:3], new_zooms=zooms)
-#     ct_from, affine_from = reslice(
-#         ct_from, affine=CT_orig.affine,
-#         zooms=CT_orig.header.get_zooms()[:3], new_zooms=zooms)
-#
-#     # first optimize the translation on the zoomed images using
-#     # ``factors`` which looks at the image at different scales
-#     reg_affine = affine_registration(
-#         moving=ct_from,
-#         static=mri_to,
-#         moving_affine=affine_from,
-#         static_affine=affine_to,
-#         nbins=32,
-#         metric='MI',
-#         pipeline=[center_of_mass, translation],
-#         level_iters=[100, 100, 10],
-#         sigmas=[3.0, 1.0, 0.0],
-#         factors=[4, 2, 1])[1]
-#
-#     CT_translated = resample(moving=CT_orig.get_fdata(),
-#                              static=T1.get_fdata(),
-#                              moving_affine=CT_orig.affine,
-#                              static_affine=T1.affine,
-#                              between_affine=reg_affine)
-#
-#     # Now, fine-tune the registration
-#     reg_affine = affine_registration(
-#         moving=CT_translated.get_fdata(),
-#         static=T1.get_fdata(),
-#         moving_affine=CT_translated.affine,
-#         static_affine=T1.affine,
-#         nbins=32,
-#         metric='MI',
-#         pipeline=[rigid],
-#         level_iters=[100, 100, 10],
-#         sigmas=[3.0, 1.0, 0.0],
-#         factors=[4, 2, 1])[1]
-#
-#     CT_aligned = resample(moving=CT_translated.get_fdata(),
-#                           static=T1.get_fdata(),
-#                           moving_affine=CT_translated.affine,
-#                           static_affine=T1.affine,
-#                           between_affine=reg_affine)
-
-
-###############################################################################
-# The previous section takes several minutes to execute so the results are
-# presented here pre-computed for convenience.
-
-alignment_affine = np.array([
-    [0.99235816, -0.03412124, 0.11857915, -133.22262329],
-    [0.04601133, 0.99402046, -0.09902669, -97.64542095],
-    [-0.11449119, 0.10372593, 0.98799428, -84.39915646],
```

Comment:
@alexrockhill okay if I just revert the text removal here, then we're good to merge?

Reply:
That's the correct alignment but the commented out code should return that matrix and it doesn't right?

Reply:
By revert the text I mean both this affine and the old commented code, which I assume is what you used to generate it